packages/documentation/src/stories/components/cards/product-card/product-card.docs.mdx

```diff
@@ -0,0 +1,51 @@
+import { Canvas, Controls, Meta, Source } from '@storybook/blocks';
+import { PostAlert, PostTabHeader, PostTabPanel, PostTabs } from '@swisspost/design-system-components-react';
+import * as ProductCardStories from './product-card.stories';
+import StylesPackageImport from '../../../../shared/styles-package-import.mdx';
+import ProductCardSyncHeights from './product-card.sample.js?raw';
+import ProductCardAngularSyncHeights from './product-card.sample.ts?raw';
+
+<Meta of={ProductCardStories} />
+
+# Product Card
+
+
+<div className="lead">
+  Highlight essential product details and provide an entry point to access in-depth information.
```

Comment:
```suggestion
  An highlight essential product details and an entry point to access in-depth information.
```
Maybe something more like this?

Reply:
Usually we write leads with action verbs. Here are some examples: 
- Accordion: "Toggle the visibility of a set of related content in your project."
- Button: "Use our custom button styles for actions in forms, dialogs, and more with support for multiple sizes, states, and more."
- Modal: "Display information that requires the user’s immediate attention."
- Toast: "Push notifications to your visitors with a .toast element, a lightweight and easily customizable alert message."

Reply:
Yes you are right, I misread "Highlight" as a verb that's why I was confused after.